client/me/purchases/cancel-purchase/refund-information.jsx

```diff
@@ -284,45 +360,61 @@ const CancelPurchaseRefundInformation = ( {
 		includedDomainPurchase &&
 		isDomainMapping( includedDomainPurchase )
 	) {
-		text = i18n.translate(
-			'This plan includes the custom domain mapping for %(mappedDomain)s. ' +
-				'The domain will not be removed along with the plan, to avoid any interruptions for your visitors. ',
-			{
-				args: {
-					mappedDomain: includedDomainPurchase.meta,
-					mappingCost: includedDomainPurchase.priceText,
-				},
-			}
+		text.push(
+			i18n.translate(
+				'This plan subscription includes the custom domain, %(domain)s' +
+					'{{strong}}The domain will not be removed{{/strong}} along with the plan, to avoid any interruptions for your visitors. ',
```

Comment:
Is this missing a period between the two sentences?

Reply:
Copy updated